includes/frontend/class-actions-listener.php

```diff
@@ -300,6 +320,26 @@ public function maybe_create_user() {
 							// User is forced to set up account with email sent to them. This password will remain a secret.
 							$password = wp_generate_password();
 							$notify   = 'both';
+
```

Comment:
When you already have the comment for the hook please use phrases like:

/** This filter is documented in includes/core/class-blocks.php (proper filename) */
/** This action is documented in includes/core/class-blocks.php (proper filename) */